### PR TITLE
Update the command of install jars into local repository quickly.

### PR DESCRIPTION
**Pull request by chenrujun**
As title

### CODE REVIEW

File: sdk/spring/README.md [hunk 1]
```diff
@@ -19,23 +19,24 @@ You can use the following command to install jars into your local repository qui
...
   -Pdev \
   -T 4 \
+  -ntp \
   -o \
-  -Dcheckstyle.skip \
-  -Dcodesnippet.skip \
-  -Denforcer.skip \
-  -Djacoco.skip \
-  -Dmaven.javadoc.skip \
-  -Dmaven.test.skip \
```
Comment (chenrujun) [inline sdk/spring/README.md:31]: `maven.test.skip` can not be used because we need to build test-jar for spring-messaging-core.

Comment (chenrujun) [reply sdk/spring/README.md:31]: `Dmaven.test.skip=true` will skip compiling the tests.

Refs: https://maven.apache.org/surefire/maven-surefire-plugin/examples/skipping-tests.html

**Review by saragluna — CHANGES_REQUESTED** [review]
please create a pr to this PR https://github.com/Azure/azure-sdk-for-java/pull/27951 instead.

File: sdk/spring/README.md (hunk 1 above)
Comment (saragluna) [reply sdk/spring/README.md:31]: but why do we need to build the test jar for users' local development?

**Comment by chenrujun** [comment]
Closing in favor of https://github.com/saragluna/azure-sdk-for-java/pull/8

File: sdk/spring/README.md (hunk 1 above)
Comment (chenrujun) [reply sdk/spring/README.md:31]: [spring-messaging-azure-eventhubs](https://github.com/Azure/azure-sdk-for-java/blob/14b39e6219fc3ef1ea02367de1e636d72f6f803f/sdk/spring/spring-messaging-azure-eventhubs/pom.xml#L46) depends on the test-jar.

**Pull request closed without merging**